code/modules/paperwork/pen.dm

```diff
@@ -147,8 +147,26 @@
 	var/on = FALSE
 	var/brightness_on = 2
 	light_color = LIGHT_COLOR_RED
+	var/backstab_sound = 'sound/items/unsheath.ogg'
 	armour_penetration_flat = 20
 
+/obj/item/pen/edagger/attack(mob/living/M, mob/living/user, def_zone)
+	force = 18
+	if(user.dir == M.dir && !HAS_TRAIT(M, TRAIT_FLOORED) && user != M)
+		force += 12
```

Comment:
I dont like setting the force to 18 here. 

I am pretty sure htis means if you bap people with the pen in pen mode it will brutally hit them for 18 damage.

what if you had a var on the proc that was like `var/applied_extra_force = FALSE` then set it to true when you increase the force. then check after the parent call if you increased the force, then decrease it.

also please make the `12` a variable. like `backstab damage`